libs/preprocessor.py

```
@@ -0,0 +1,80 @@
+            else:
+                raise Exception(f"Failed to download the file. Status code: {response.status_code}")
+
+        except Exception as e:
+            c.s.result = False
+            exc_type, value, tb = sys.exc_info()
+            exception_details = (f"Exception Type: {exc_type} Exception value: {value} "
+                                 f"at line no: {tb.tb_lineno}")
+            print(exception_details)
+            raise Exception() from e
+        finally:
+            c.update
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)